src/Agent.Worker/Build/TFCommandManager.cs

```diff
@@ -48,7 +48,7 @@ public string ResolvePath(string serverPath)
             return localPath?.Trim() ?? string.Empty;
         }
 
-        public async Task ScorchAsync() => await RunCommandAsync(FormatFlags.OmitCollectionUrl, "vc", "scorch", "/diff", "/recursive", SourcesDirectory);
+        public async Task ScorchAsync() => await RunCommandAsync(FormatFlags.OmitCollectionUrl, "vc", "scorch", "/diff", "/recursive", "/unmapped", SourcesDirectory);
```

Comment:
The `/unmapped` switch is simply an enhancement. I hadn't noticed this switch before (or it wasn't there?). It instructs scorch to traverse cloaked directories and traverse subdirectories for single-level maps.